03week/airplane/css/style.css

```diff
@@ -0,0 +1,171 @@
+  /*reset all margins and paddings to zero*/
+  *{
+      margin: 0;
+      padding: 0;
+    }
+
+    .main {
+      background: url(../img/airline.jpg);
+      background-size: cover;
+      background-repeat: no-repeat;
+      height: 700px;
+      width: 100%;
+      margin-top:0;
+      border-style: solid;
+    }
+
+    .header {
+      height: 50px;
+      background-color:#324a70;/*navbar header*/
+      width: 100%;
+    }
+    /*header image*/
+    .header img {
+      height: 50px;
+    }
+
+    .header .navbar {
+    list-style-type: none;
+    float: right;
+    margin-top: 12px;
+    margin-right: 60px;
+    }
+
+    .header .navbar a {
+      padding:10px;
+      text-decoration: none;
+      font-family: 'Vollkorn', serif;
+      color: white;
+    }
+    /*navbar icons*/
+    .header .navbar img {
+      height: 13px;
+      margin-right: 7px;
+    }
+
+  /*navbar icon enlarges and changes color on hover*/
+    .header .navbar img:hover {
+      height: 20px;
+      background-color: #14d18c;
+    }
+    /*On hover navbar selection changes color*/
+    .header .navbar a:hover {
+      font-size: 20px;
+      font-weight: bold;
+      font-family: 'Shadows Into Light', cursive;
+      color: #14d18c;
+    }
+
+    .main .planner {
+      height: 400px;
+      margin-top:100px;
+      margin-right: 872px;
+      margin-left: 23px;
+      border-radius: 2%;
+      background-color: #324a70;
+    }
+
+    .planner .btn-group {
+      height: 40px;
+      background-color: #046266;
+    }
+
+  /*images:plane-icon|hotel-icon|car-icon*/
+    .planner .btn-group img {
+      height: 38px;
+      margin-top: 5px;
+      margin-bottom:5px;/*i don´t under why this declaration is not working*/
```

Comment:
this is working, but it is not pushing down the btn-group div.  If you set the height taller on btn-group you will get the spacing